content/md/en/docs/fundamentals/light-clients-in-substrate-connect.md

```diff
@@ -0,0 +1,273 @@
+const api = await ApiPromise.create({ provider });
+await api.rpc.chain.subscribeNewHeads((lastHeader) => {
+console.log(lastHeader.hash);
+});
+await api.disconnect();
+```
+
+## Use Substrate Connect with other libraries
+
+The previous section demonstrated how to add the Substrate Connect provider to applications that use the Polkadot-JS API. 
+With this provider, you create applications that enable users to interact with the chain through the browser using calls to the Polkadot-JS API methods. 
+However, you can install and use @substrate-connect in applications that don't depend on the Polkadot-JS API. For example, if you are building your own application library or programming interfaces, you can install the Substrate Connect dependencies by running the appropriate command for the package manager you use.
+
+For example, if you use `yarn`, run the following command:
+
+```bash
+yarn add @substrate/connect
+```
+
+If you use `npm` as your package manager, run the following command:
+```bash
+npm i @substrate/connect
+```
+
+### Connect to a well-known chain
+
+The following example illustrates how you can use Substrate Connect to connect to a well-known network such as  Polkadot, Kusama, Westend, or Rococo.
+
+```js
+import { WellKnownChain, createScClient } from '@substrate/connect';
+// Create the client
+const client = createScClient();
+// Create the chain connection, while passing the `jsonRpcCallback` function.
+const chain = await client.addWellKnownChain(
+WellKnownChain.polkadot,
+function jsonRpcCallback(response) {
+console.log('response', response);
+}
+);
+// send a RpcRequest
+chain.sendJsonRpc(
+'{"jsonrpc":"2.0","id":"1","method":"system_health","params":[]}'
+);
+```
+
+### Connect to a parachain
+
+The following example illustrates how you can use Substrate Connect to connect to a parachain.
+
+```js
+import { WellKnownChain, createScClient } from '@substrate/connect';
+import jsonParachainSpec from './jsonParachainSpec.json';
+// Create the client
+const client = createScClient();
+// Create the relay chain connection. There is no need to pass a callback
+// function because we will sending and receiving messages through
+// the parachain connection.
+await client.addWellKnownChain(WellKnownChain.westend2);
+// Create the parachain connection.
+const parachainSpec = JSON.stringify(jsonParachainSpec);
+const chain = await client.addChain(
+parachainSpec,
+function jsonRpcCallback(response) {
+console.log('response', response);
+}
+);
+// send a request
+chain.sendJsonRpc(
+'{"jsonrpc":"2.0","id":"1","method":"system_health","params":[]}'
+);
+```
+
+## API Documentation
+
+For more information about the substrate-connect API, see [Substrate Connect](https://paritytech.github.io/substrate-connect/api/).
+
+## Browser extension
+
+Substrate Connect provides a browser extension for the @substrate/light client node that allows the light client to run inside the extension instead of within the context of every browser tab. 
```

Comment:
```suggestion
Substrate Connect provides a browser extension for the @substrate/light client node that allows the Smoldot light client to run inside the extension instead of within the context of every browser tab. 
```